playbooks/common/openshift-cluster/config.yml

```diff
@@ -42,6 +42,14 @@
     r_openshift_excluder_action: disable
     r_openshift_excluder_service_type: "{{ openshift.common.service_type }}"
 
+- name: Disable excluders
+  hosts: oo_masters_to_config:oo_nodes_to_config
+  tasks:
+  - name: Set hostname
+    command: >
```

Comment:
Would hostname module be better than command here?

http://docs.ansible.com/ansible/latest/hostname_module.html